modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -223,7 +297,12 @@ private Configuration _createConfiguration(
 					Constants.SERVICE_PID, configurationPid));
 
 		try {
-			Files.write(_configurationPath, content.getBytes());
+			if (unsafeRunnable == null) {
+				Files.write(_configurationPath, content.getBytes());
```

Comment:
@mtambara this is still very confusing. The String.getBytes() here uses system default charset for encoding. You should just change it to use String.getBytes(Charset) or String.getBytes(String) to force an explicit charset, rather than doing this UnsafeRunnable thing.

Also the variable names is confusing too, utfLine/isoLine. Once in memory as String, there are just chars, no encoding info. Encoding only exist in bytes, representing how we converted the chars to bytes. Please sort these out.